src/sonic-yang-models/yang-models/sonic-copp.yang

```diff
@@ -133,6 +138,16 @@ module sonic-copp {
 					default "forward";
 					description "Red action";
 				}
+
```

Comment:
These are internal fields and shouldn't be user configurable. Please check the comment here https://github.com/sonic-net/sonic-buildimage/pull/7199#discussion_r610339266